css/properties/grid-row-end.json

```diff
@@ -105,9 +105,14 @@
             "safari_ios": {
               "version_added": "10.3"
             },
-            "samsunginternet_android": {
-              "version_added": "6.0"
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "6.0"
```

Comment:
This can probably also be reverted since I think the version_added: true is from a flag.